signer/Cargo.toml

```diff
@@ -16,7 +16,8 @@ doppelganger_protection = { workspace = true }
 futures = { workspace = true }
 helper_functions = { workspace = true }
 itertools = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order